pkgs/development/python-modules/geopy/default.nix

```diff
@@ -1,31 +1,33 @@
 { stdenv
 , buildPythonPackage
-, fetchPypi
-, isPy27
+, fetchFromGitHub
+, isPy3k
 , mock
 , tox
 , pylint
+, geographiclib
 }:
 
 buildPythonPackage rec {
-  pname = "geopy";
-  version = "1.20.0";
-  disabled = !isPy27;
+    pname = "geopy";
+    version = "2.0-unstable-2019-11-10";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "9419bc90ee6231590c4ae7acf1cf126cefbd0736942da7a6a1436946e80830e2";
-  };
+    disabled = !isPy3k; # only Python 3
+    doCheck = false; # too many checks
 
-  doCheck = false;  # too much
+    buildInputs = [ mock tox pylint ];
```

Comment:
```suggestion
```

They're only needed for the tests and since we don't run them ...
Please also remove in the derivation arguments and the other version.

Reply:
Can't I just add them in `checkInputs` in case `doCheck` gets changed in the future ?

Reply:
Hmm, you could. But it would likely be out of date then.
It looks out of date even now: https://github.com/geopy/geopy/blob/master/setup.py

So I'd just remove it.
Also do you expect them to remove the dependency on network access? Looks like this package needs network access for everything :laughing: